From: Reto Buerki <reet@codelabs.ch>
Date: Fri, 21 Mar 2014 13:29:06 +0000
Subject: Disable multicast tests
 These tests do not work on the buildd's.
Forwarded: not-needed

--- a/tests/socket_tests-ip.adb
+++ b/tests/socket_tests-ip.adb
@@ -158,12 +158,12 @@ package body Socket_Tests.IP is
       T.Add_Test_Routine
         (Routine => Send_V6_Datagram'Access,
          Name    => "Send data (IPv6, datagram)");
-      T.Add_Test_Routine
-        (Routine => Send_Multicast_V4'Access,
-         Name    => "Send data (IPv4, multicast)");
-      T.Add_Test_Routine
-        (Routine => Send_Multicast_V6'Access,
-         Name    => "Send data (IPv6, multicast)");
+--        T.Add_Test_Routine
+--          (Routine => Send_Multicast_V4'Access,
+--           Name    => "Send data (IPv4, multicast)");
+--        T.Add_Test_Routine
+--          (Routine => Send_Multicast_V6'Access,
+--           Name    => "Send data (IPv6, multicast)");
       T.Add_Test_Routine
         (Routine => Listen_Callbacks'Access,
          Name    => "Data reception callback handling");
